jobs/jobs.go

```diff
@@ -101,8 +101,10 @@ func (srv *JobServer) SetJobError(job *model.Job, jobError *model.AppError) *mod
 	if job.Data == nil {
 		job.Data = make(map[string]string)
 	}
-	job.Data["error"] = jobError.Message + " — " + jobError.DetailedError
-
+	job.Data["error"] = jobError.Message
+	if len(jobError.DetailedError) > 0 {
+		job.Data["error"] += " — " + jobError.DetailedError
```

Comment:
Out of curiosity (and not required but I thought I mention) , would using fmt.sprintf be better or not? I guess it's preferential since not a lot of string concatenation but I thought I mention it regardless.